ansible/roles_ocp_workloads/ocp4_workload_cyberark_dap/templates/secrets_cluster_role.j2

```diff
@@ -0,0 +1,8 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: secrets-access
+rules:
+  - apiGroups: [""]
+    resources: ["secrets"]
+    verbs: [ "get", "update" ]
```

Comment:
Also here, please use regular YAML rather than inline JSON style

Reply:
And also here. TY!